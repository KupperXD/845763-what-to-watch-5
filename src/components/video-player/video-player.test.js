import React from 'react';
import renderer from 'react-test-renderer';
import {BrowserRouter} from "react-router-dom";

import VideoPlayer from "./video-player";


it(`renders correctly`, () => {
  const videoPlayer = renderer
    .create(
        <BrowserRouter>
          <VideoPlayer
            isPlaying={true}
            muted={true}
            picture={``}
            src={``}
          />
        </BrowserRouter>
    )
    .toJSON();

  expect(videoPlayer).toMatchSnapshot();
});
