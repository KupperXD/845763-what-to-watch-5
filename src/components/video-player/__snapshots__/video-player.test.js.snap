// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<video
  autoPlay={true}
  controls={false}
  muted={true}
  poster=""
  style={
    Object {
      "maxWidth": "100%",
      "objectFit": "cover",
    }
  }
>
  <source
    src=""
    type="video/"
  />
</video>
`;
